twine/package.py

```diff
@@ -21,9 +21,12 @@
 import pkg_resources
 
 try:
-    import pyblake2
+    from hashlib import blake2b as hashlib_blake2b
```

Comment:
Sure. I did this only because I figured it'd make it slightly easier to switch to just import hashlib in the future. Will adjust when I get back to my computer. 

Sent from my iPhone

> On Sep 7, 2016, at 7:25 AM, Ian Cordasco <notifications@github.com> wrote:
> 
> In twine/package.py:
> 
> > @@ -21,9 +21,12 @@
> >  import pkg_resources
> >  
> >  try:
> > -    import pyblake2
> > +    from hashlib import blake2b as hashlib_blake2b
> Can we just do:
> 
> try:
>     from hashlib import blake2b
> except ImportError:
>     try:
>         from pyblake2 import blake2b
>     except ImportError:
>         blake2b = None
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>